src/Microsoft.TemplateEngine.Utils/CryptoRandom.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Security.Cryptography;
+
+namespace Microsoft.TemplateEngine.Utils
+{
+    public class CryptoRandom : Random
```

Comment:
Is there a use for inheriting from `Random` today or is this more for the future use cases?

Reply:
It was for convenience of swapping current usages of `Random`
But since current usages use just a single function anyway - I've removed this